openmdao/docs/features/building_blocks/components/external_code_implicit_comp.rst

```diff
@@ -12,29 +12,28 @@ operating system. But it treats the `Component` as an `ImplicitComponent` rather
 
```

Comment:
Seems like we need a little more in the docs since ExternalCodeImplicitComp now differs from ExternalCodeComp with respect to having the two commands versus the one?

Reply:
New docs, all comments addressed.